backend/app/services/chat/model_resolver.py

```
@@ -372,40 +395,60 @@ def _extract_model_config(model_spec: Dict[str, Any]) -> Dict[str, Any]:
     if not default_headers:
         # settings.EXECUTOR_ENV is a JSON string, need to parse it first
         try:
-            executor_env = json.loads(settings.EXECUTOR_ENV) if settings.EXECUTOR_ENV else {}
+            executor_env = (
+                json.loads(settings.EXECUTOR_ENV) if settings.EXECUTOR_ENV else {}
+            )
             default_headers = executor_env.get("DEFAULT_HEADERS", {})
         except (json.JSONDecodeError, TypeError):
-            logger.warning(f"[model_resolver] Failed to parse settings.EXECUTOR_ENV as JSON")
+            logger.warning(
+                f"[model_resolver] Failed to parse settings.EXECUTOR_ENV as JSON"
```

Ruff (0.14.7)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)